example/image-classification/symbols/densenet.py

```diff
@@ -0,0 +1,182 @@
+    units_num : int
+        the number of BasicBlock in each DenseBlock
+    data : str
+        Input data
+    growth_rate : int
+        Number of output channels
+    drop_out : float
+        Probability of an element to be zeroed. Default = 0.2
+    workspace : int
+        Workspace used in convolution operator
+    """
+
+    for i in range(units_num):
+        Block = BasicBlock(data, growth_rate=growth_rate, stride=(1, 1), name=name + '_unit%d' % (i + 1),
+                           bottle_neck=bottle_neck, drop_out=drop_out,
+                           bn_mom=bn_mom, workspace=workspace)
+        data = mx.symbol.Concat(data, Block, name=name + '_concat%d' % (i + 1))
+    return data
+
+def TransitionBlock(num_stage, data, num_filter, stride, name, drop_out=0.0, bn_mom=0.9, workspace=512):
+    """Return TransitionBlock Unit symbol for building DenseNet
+    Parameters
+    ----------
+    num_stage : int
+        Number of stage
+    data : str
+        Input data
+    num : int
+        Number of output channels
+    stride : tupe
+        Stride used in convolution
+    name : str
+        Base name of the operators
+    drop_out : float
+        Probability of an element to be zeroed. Default = 0.2
+    workspace : int
+        Workspace used in convolution operator
+    """
+    bn1 = mx.sym.BatchNorm(data=data, fix_gamma=False, eps=2e-5, momentum=bn_mom, name=name + '_bn1')
+    act1 = mx.sym.Activation(data=bn1, act_type='relu', name=name + '_relu1')
+    conv1 = mx.sym.Convolution(data=act1, num_filter=num_filter,
+                               kernel=(1, 1), stride=stride, pad=(0, 0), no_bias=True,
+                               workspace=workspace, name=name + '_conv1')
+    if drop_out > 0:
+        conv1 = mx.symbol.Dropout(data=conv1, p=drop_out, name=name + '_dp1')
+    return mx.symbol.Pooling(conv1, global_pool=False, kernel=(2, 2), stride=(2, 2), pool_type='avg',
+                             name=name + '_pool%d' % (num_stage + 1))
+
+def get_symbol(num_classes, num_layers=121, reduction=0.5, drop_out=0.2, bottle_neck=True, bn_mom=0.9, 
+                            workspace=512, num_stage=4, growth_rate=32,  **kwargs):
+    """
+    Adapted from https://github.com/bruinxiong/densenet.mxnet/blob/master/symbol_densenet.py
+    Return DenseNet symbol of imagenet
+    Parameters
+    ----------
+    num_class : int
+        Ouput size of symbol
+    num_layers : int
+        Number of layers of the whole net
+    reduction : float
+        Compression ratio. Default = 0.5
+    drop_out : float
+        Probability of an element to be zeroed. Default = 0.2
+    workspace : int
+        Workspace used in convolution operator
+    num_stage : int
+        Number of stage
+    growth_rate : int
+        Number of output channels
+    """
+
+    if num_layers == 121:
+        units = [6, 12, 24, 16]
+    elif num_layers == 169:
+        units = [6, 12, 32, 32]
+    elif num_layers == 201:
+        units = [6, 12, 48, 16]

```

Comment:
To be consistent with https://arxiv.org/pdf/1608.06993.pdf you would want:
```
elif num_layers == 201:
  units = [6, 12, 48, 32]
```
I dont know if this is a typo here or in the paper.